<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLoadVolume</class>
 <widget class="QDialog" name="DialogLoadVolume">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>377</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000000</width>
    <height>1000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Load Volume</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select volume file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBoxFilenames">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOpen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxResampleToRAS">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Resample to standard RAS space</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxAsVector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Load as vector</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxRegistration">
     <property name="text">
      <string>Apply registration file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditRegistration">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOpenRegistration">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="title">
      <string notr="true">Sample method</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioNearest">
        <property name="text">
         <string>Nearest neighbor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioTrilinear">
        <property name="text">
         <string>Trilinear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>115</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ALWAYS choose 'Nearest Neighbor' if you are loading a label volume.</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioCubic">
        <property name="text">
         <string>Cubic</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Color map</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxColorMap">
       <item>
        <property name="text">
         <string extracomment="grayscale">Grayscale</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lookup Table</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Heat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Turbo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GE Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NIH</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelLUT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Look up table</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxLUT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="freeview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogLoadVolume</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRegistration</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditRegistration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRegistration</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonOpenRegistration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpen</sender>
   <signal>clicked()</signal>
   <receiver>DialogLoadVolume</receiver>
   <slot>OnOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpenRegistration</sender>
   <signal>clicked()</signal>
   <receiver>DialogLoadVolume</receiver>
   <slot>OnOpenRegistration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxColorMap</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogLoadVolume</receiver>
   <slot>OnColorMap(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxLUT</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogLoadVolume</receiver>
   <slot>OnLUT(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogLoadVolume</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOpen()</slot>
  <slot>OnOpenRegistration()</slot>
  <slot>OnColorMap(int)</slot>
  <slot>OnLUT(int)</slot>
  <slot>OnOK()</slot>
 </slots>
</ui>
